source/decryptor/decryptor.c

```diff
@@ -113,7 +132,74 @@ u32 DecryptTitlekeys(void)
     }
     FileClose();
 
-    Debug("Done!");
```

Comment:
This message is redundant. The success message is given after completion of each feature function anyways.